AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -74,6 +75,8 @@ @implementation MSDataStore
 - (instancetype)init {
   if ((self = [super init])) {
     _tokenExchangeUrl = (NSURL *)[NSURL URLWithString:kMSDefaultApiUrl];
+
+    // FIXME: move that to document store (task #60212)
```

Comment:
That's not going to be very easy. This would require every method in the document store to become asynchronous. That's why I kept it here.

If not done already, I suggest you wrap the entire sync call in a block to run it on the dispatch queue.

Reply:
it's resolved